<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VistaLoginCliente</class>
 <widget class="QWidget" name="VistaLoginCliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login Cliente - CineMax</string>
  </property>
  <widget class="QLabel" name="labelTitolo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Variable</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #D7AA0C;
}</string>
   </property>
   <property name="text">
    <string>Login Cliente</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAccedi">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>310</y>
     <width>91</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #963C00;
    color: #FF7800;
    border: 2px solid #502800;
    border-radius: 6px;
}

QPushButton:focus {
    outline: none;
}

QPushButton:hover {
    background-color: #C86400;
}
</string>
   </property>
   <property name="text">
    <string>Accedi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRegistrati">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>310</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #963C00;
    color: #FF7800;
    border: 2px solid #502800;
    border-radius: 6px;
}

QPushButton:focus {
    outline: none;
}

QPushButton:hover {
    background-color: #C86400;
}
</string>
   </property>
   <property name="text">
    <string>Registrati</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditEmail">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>200</y>
     <width>231</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #321E00;
    color: #965A00;
    border: 1px solid #190E00;
    border-radius: 4px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPassword">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>270</y>
     <width>231</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #321E00;
    color: #965A00;
    border: 1px solid #190E00;
    border-radius: 4px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelEmail">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>170</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #C87800;
}</string>
   </property>
   <property name="text">
    <string>Email:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPassword">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #C87800;
}</string>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRecuperoPassword">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>450</y>
     <width>161</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #963C00;
    color: #FF7800;
    border: 2px solid #502800;
    border-radius: 6px;
}

QPushButton:focus {
    outline: none;
}

QPushButton:hover {
    background-color: #C86400;
}
</string>
   </property>
   <property name="text">
    <string>Recupero Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="Sfondo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>501</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background: qlineargradient(
        x1: 0, y1: 1,
        x2: 0, y2: 0,
		stop: 0 #211400,
        stop: 1 #7D4B00
    );
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="ClickableLabel" name="labelIndietroButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>20</y>
     <width>63</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Immagini/IndietroButtonCliente.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelErroreEmail">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>200</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FFFF00;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelErrorePassword">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>270</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FFFF00;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelRecuperoPassword">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>450</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FFFF00;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <zorder>Sfondo</zorder>
  <zorder>labelTitolo</zorder>
  <zorder>pushButtonAccedi</zorder>
  <zorder>pushButtonRegistrati</zorder>
  <zorder>lineEditEmail</zorder>
  <zorder>lineEditPassword</zorder>
  <zorder>labelEmail</zorder>
  <zorder>labelPassword</zorder>
  <zorder>pushButtonRecuperoPassword</zorder>
  <zorder>labelIndietroButton</zorder>
  <zorder>labelErroreEmail</zorder>
  <zorder>labelErrorePassword</zorder>
  <zorder>labelRecuperoPassword</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>clickablelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
